mx.sulong/mx_sulong.py

```diff
@@ -1111,6 +1115,26 @@ def checkNoHttp(args=None):
                 exit(-1)
             line_number += 1
 
+def genInlineAssemblyParser(args=None, out=None):
+    """generate inline assembly parser and scanner if corresponding grammer is new"""
+    generatedParserDir = _inlineAssemblySrcDir + _inlineAssemblyPackageName.replace('.', '/')
+    generatedParserFile = generatedParserDir + "/Parser.java"
+    generatedScannerFile = generatedParserDir + "/Scanner.java"
+    if not isfile(generatedParserFile) or not isfile(generatedScannerFile) or os.path.getmtime(_inlineAssemblyGrammer) >= os.path.getmtime(generatedParserFile):
+        localCocoJarFile = _suite.dir + "/lib/Coco.jar"
+        if not isfile(localCocoJarFile):
+            jarFileUrls = ["http://www.ssw.uni-linz.ac.at/Coco/Java/Coco.jar"]
+            mx.download(localCocoJarFile, jarFileUrls)
+        command = [mx.get_jdk(tag='jvmci').java, "-jar", localCocoJarFile, "-package", _inlineAssemblyPackageName, "-o", generatedParserDir, _inlineAssemblyGrammer]
+        mx.run(command)
+        #Files get generated in Windows file format. Convert them to avoid style check failure during regression testing
+        mx.run(['dos2unix', generatedParserFile, generatedScannerFile])
+
+def sulongBuild(args=None, out=None):
+    """custom build command to wrap inline assembly parser generation"""
+    genInlineAssemblyParser()
+    originalBuildCommand(['-p', '--warning-as-error', '--no-native', '--force-javac'])
```

Comment:
Why do we need the parameters? I do not think that we want to force users to use javac. Can we just pass `args` to `originalBuildCommand`?